PocketKit/Sources/PocketKit/PocketNotificationService.swift

```diff
@@ -167,3 +213,19 @@ extension PocketNotificationService: BrazeInAppMessageUIDelegate {
         // Executed when `message` is presented to the user
     }
 }
+
+/**
+ V3 Helpers
+ */
+extension PocketNotificationService {
+
+    static func pushType() -> PushType {
```

Comment:
This can be a property rather than a function since it's not really performing any function; rather, it's defining a property of the service.